invokeai/app/services/graph.py

```diff
@@ -773,6 +773,24 @@ class GraphExecutionState(BaseModel):
         default_factory=dict,
     )
 
+    # Declare all fields as required; necessary for OpenAPI schema generation build.
```

Comment:
Do you only need to be marking fields with `default_factory` here?

Reply:
Technically, yes. Would you like to exclude the field that doesn't have a `default_factory`? I suppose that's the right thing to do.

Reply:
Hmm. Actually, the generated schema for `graph`, which does not have a `default_factory`, is marked optional unless I include it here. So I believe this is correct insofar as it is needed for the generation to work correctly.